www/modules/users/controllers/settings.client.controller.es6

```diff
@@ -1,37 +1,33 @@
 class SettingsController {
-  static get $inject() {
-    return ['$injector'];
-  }
-
-  constructor($injector) {
-    this.Authentication = $injector.get('Authentication');
-    this.$location      = $injector.get('$location');
-    if (!this.Authentication.user) this.$location.path('/');
+  /*@ngInject*/
+  constructor(Authentication, Notification, Users, FileUploader, MIHUtils, $timeout, $window, $rootScope, $location) {
+    Object.assign(this, {
+      Authentication,
+      Notification,
+      Users,
+      FileUploader,
+      MIHUtils,
+      $timeout,
+      $window,
+      $rootScope,
+      $location
+    });
 
-    /*@ngInject*/
-    this.Users        = $injector.get('Users');
-    this.user         = this.Authentication.user;
-    this.imageURL     = this.user.profileImageURL;
-    this.$timeout     = $injector.get('$timeout');
-    this.$window      = $injector.get('$window');
-    this.FileUploader = $injector.get('FileUploader');
-    // TODO: move to root controller + share
-    // TODO: refactor into directive so that we do not need to always inject it
-    this.MIHUtils     = $injector.get('MIHUtils');
-    this.$rootScope   = $injector.get('$rootScope');
-    this.Notification = $injector.get('Notification');
+    if (!this.Authentication.user) {
+      this.$location.path('/');
+    }
 
-    /*fields*/
+    this.user        = this.Authentication.user;
+    this.imageURL    = this.user.profileImageURL;
     this.workingDays = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
     this.reminders   = [5, 10, 15, 20, 25, 30];
```

Comment:
We don't have reminder functionality. Could you please remove redundant dead code?